AdSecGH/Helpers/BusinessExtensions.cs

```diff
@@ -109,6 +130,36 @@ private static readonly Dictionary<Type, Func<Attribute, object>> ToGoo
         },
       };
 
+    private static readonly Dictionary<Type, Func<object, object>> GooToParam
+      = new Dictionary<Type, Func<object, object>> {
+        {
+          typeof(LengthParameter), goo => {
+            return UnitHelpers.ParseToQuantity<Length>(goo, DefaultUnits.LengthUnitGeometry);
+          }
+        }, {
+          typeof(AdSecSectionParameter), goo => {
+            dynamic gooDynamic = goo;
+            return new AdSecSectionGoo(gooDynamic);
+          }
+        }, {
+          typeof(DoubleParameter), goo => {
+            if (goo is double value) {
+              return value;
+            }
+
+            return null;
+          }
+        },
+      };
+
+    public static void UpdateProperties(this IFunction BusinessComponent, GH_Component component) {
```

Comment:
not sure it should be here

Reply:
anything that is part of IFunction but needs access to GH I've put there.